scylladb/cpp-driver
# Add Scylla Sphinx Theme 1.0

Replaces https://github.com/scylladb/scylla-operator/pull/802

Sphinx ScyllaDB Theme 1.0 is now released 🥳 

We’ve made a number of updates to the look and feel of the theme to improve the overall user experience.
You can read more about all notable changes [here](https://sphinx-theme.scylladb.com/stable/CHANGELOG#september-2021).

This PR also cleans the file ``conf.py``, removing several unsued options.

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser. You will see the docs with the new look and feel:

![image](https://user-images.githubusercontent.com/9107969/138131190-0678c8bd-9280-40cf-bc0b-be25d0e11c1e.png)
